OpenBCI_32bit_Library.cpp

```diff
@@ -932,7 +1007,7 @@ void OpenBCI_32bit_Library::processIncomingLeadOffSettings(char character) {
         numberOfIncomingSettingsProcessedLeadOff = 0;
 
         // put flag back down
-        isProcessingIncomingSettingsLeadOff = false;
+        endMultiCharCmdTimer();
 
```

Comment:
do you mind removing this space?

Reply:
Not sure if I understand which space - just check it is right

Reply:
Line 1011, looks like there was an extra space in this function already, was just wondering since your here, if you mind removing the space below your addition.

Reply:
Got it!